### DISCUSSION TITLE
[Feature Request]Show different taskbar icon for different output device

### DISCUSSION THREAD

**Discussion by Lucifer050a** [File-New-Project/EarTrumpet#866] [Ideas]
When change output device, also change the taskbar icon.

The icon can customize or program built-in (like heeadphone, tv or speaker).

**Comment by PhilippeCadieux** [comment]
Came here to suggest the same thing. I currently use an old program called "Audio Switch" to quickly switch between my three different audio outputs (speakers, headphone or TV connected as third monitor). I just found EarTrumpet while looking for a way to quickly adjust the volume of my Line-In input (which it does) so I would love to just ditch the old program and only use EarTrumpet but without a way to quickly see at a glance what current output device is currently selected in windows then I can't really use it.

**Comment by ghost** [comment]
SoundSwitch have this feature implemented maybe u can check their code to implement this.
